ext/cl_khr_command_buffer.asciidoc

```diff
@@ -1516,6 +1523,174 @@ New errors:
+with synchronization-points in _sync_point_wait_list_ must be the same
+as _command_buffer_. The memory associated with _sync_point_wait_list_
+can be reused or freed after the function returns.
+
+_sync_point_ Returns a synchronization-point ID that identifies this particular
+command. Synchronization-point objects are unique and can be used to
+identify this command later on. _sync_point_ can be `NULL` in which case it
+will not be possible for the application to record a wait for this command to
+complete. If the _sync_point_wait_list_ and the _sync_point_ arguments are not
+`NULL`, the _sync_point_ argument should not refer to an element of the
+_sync_point_wait_list_ array.
+
+_mutable_handle_ Returns a handle to the command. Handle is unused by
+this extension and must be passed as `NULL`.
+
+{clCommandSVMMemcpyKHR} returns {CL_SUCCESS} if the function is executed
+successfully. Otherwise, it returns the errors defined by
+{clEnqueueSVMMemcpy} except:
+
+{CL_INVALID_COMMAND_QUEUE} is replaced with:
+
+* {CL_INVALID_COMMAND_QUEUE} if _command_queue_ is not `NULL`.
+
+{CL_INVALID_CONTEXT} is replaced with:
+
+* {CL_INVALID_CONTEXT} if the context associated with _command_queue_,
+  _command_buffer_, and _kernel_ are not the same.
+
+{CL_INVALID_EVENT_WAIT_LIST} is replaced with:
+
+* {CL_INVALID_SYNC_POINT_WAIT_LIST_KHR} if _sync_point_wait_list_ is `NULL` and
+  _num_sync_points_in_wait_list_ is > 0, or _sync_point_wait_list_ is not
+  `NULL` and _num_sync_points_in_wait_list_ is 0, or if
+  synchronization-point objects in _sync_point_wait_list_ are not valid
+  synchronization-points.
+
+New errors:
+
+* {CL_INVALID_COMMAND_BUFFER_KHR} if _command_buffer_ is not a valid
+  command-buffer.
+
+* {CL_INVALID_OPERATION} if _command_buffer_ has been finalized.
+
+* {CL_INVALID_VALUE} if _mutable_handle_ is not `NULL`.
+
+
+The function
+
+include::{generated}/api/protos/clCommandSVMMemFillKHR.txt[]
+
+Records a command to fill a region in SVM with a pattern of a given pattern size.
+
+_command_buffer_ Refers to a valid command-buffer object.
+
+_command_queue_ Specifies the command-queue the command will be recorded to.
+Parameter is unused by this extension as only a single command-queue is
+supported and **must** be `NULL`.
+
+_svm_ptr_ is a pointer to a memory region that will be filled with
+_pattern_. It must be aligned to _pattern_size_ bytes.
+If _svm_ptr_ is allocated using {clSVMAlloc} then it must be allocated from
+the same context from which _command_queue_ was created.
+Otherwise the behavior is undefined.
+
+_pattern_ is a pointer to the data pattern of size _pattern_size_ in bytes.
+_pattern_ will be used to fill a region in _buffer_ starting at _svm_ptr_
+and is _size_ bytes in size.
+The data pattern must be a scalar or vector integer or floating-point data
+type supported by OpenCL as described in <<scalar-data-types, Shared
+Application Scalar Data Types>> and <<vector-data-types, Supported
+Application Vector Data Types>>.
```

Comment:
I took the lazy way out, and just removed the reference. All in all, it would be nice to remove the duplication of API descriptions here due to the "API copy".